.env-dist

```diff
@@ -34,8 +34,8 @@ TWILIO_AUTH_TOKEN=
 TWILIO_SMS_APPLICATION_SID=
 TWILIO_MAIN_NUMBER=
 PREMIUM_PROD_ID=prod_K29ULZL9pUR9Fr
-PERIODICAL_PREMIUM_PROD_ID=price_1LXUenJNcmPzuWtRw3rhjQNP
-PHONE_PROD_ID=prod_LgQiSgNi4xL7dq
+PERIODICAL_PREMIUM_PROD_ID=prod_KGizMiBqUJdYoY
```

Comment:
Production. (That's the one we want in here, right?)

Reply:
Doesn't matter too much - we'll have to override in local, dev, and stage. This is "safest" in case the `.env-dist` value is put into prod it'll still work.